sdk/include/psdk/iptypes.h

```diff
@@ -33,247 +22,306 @@
 #define MAX_DHCPV6_DUID_LENGTH          130
 #define MAX_DNS_SUFFIX_STRING_LENGTH    256
 
-#define BROADCAST_NODETYPE              1
-#define PEER_TO_PEER_NODETYPE           2
-#define MIXED_NODETYPE                  4
-#define HYBRID_NODETYPE                 8
+#define BROADCAST_NODETYPE     1
+#define PEER_TO_PEER_NODETYPE  2
+#define MIXED_NODETYPE         4
+#define HYBRID_NODETYPE        8
 
 typedef struct {
     char String[4 * 4];
 } IP_ADDRESS_STRING, *PIP_ADDRESS_STRING, IP_MASK_STRING, *PIP_MASK_STRING;
 
 typedef struct _IP_ADDR_STRING {
     struct _IP_ADDR_STRING* Next;
-    IP_ADDRESS_STRING IpAddress;
-    IP_MASK_STRING IpMask;
-    DWORD Context;
+    IP_ADDRESS_STRING       IpAddress;
+    IP_MASK_STRING          IpMask;
+    DWORD                   Context;
 } IP_ADDR_STRING, *PIP_ADDR_STRING;
 
 typedef struct _IP_ADAPTER_INFO {
     struct _IP_ADAPTER_INFO* Next;
-    DWORD ComboIndex;
-    char AdapterName[MAX_ADAPTER_NAME_LENGTH + 4];
-    char Description[MAX_ADAPTER_DESCRIPTION_LENGTH + 4];
-    UINT AddressLength;
-    BYTE Address[MAX_ADAPTER_ADDRESS_LENGTH];
-    DWORD Index;
-    UINT Type;
-    UINT DhcpEnabled;
-    PIP_ADDR_STRING CurrentIpAddress;
-    IP_ADDR_STRING IpAddressList;
-    IP_ADDR_STRING GatewayList;
-    IP_ADDR_STRING DhcpServer;
-    BOOL HaveWins;
-    IP_ADDR_STRING PrimaryWinsServer;
-    IP_ADDR_STRING SecondaryWinsServer;
-    time_t LeaseObtained;
-    time_t LeaseExpires;
+    DWORD                    ComboIndex;
```

Comment:
The indentation is because it's also present in the official MS headers?
Personally I don't think we need to use that for structure fields...